test/grpc/route_guide/get-feature.cc

```diff
@@ -0,0 +1,57 @@
+#include <string>
+
+#include "eventuals/finally.h"
+#include "eventuals/head.h"
+#include "eventuals/let.h"
+#include "eventuals/promisify.h"
+#include "eventuals/then.h"
+#include "gtest/gtest.h"
+#include "test/grpc/route_guide/route-guide-test.h"
+
+using grpc::Status;
+
+using eventuals::Finally;
+using eventuals::Head;
+using eventuals::Let;
+using eventuals::Then;
+using eventuals::operator*;
+
+TEST_F(RouteGuideTest, GetFeatureTest) {
+  auto e = [this]() {
+    return client().GetFeature(MakePoint(409146138, -746188906))
+        >> Then([this](auto&&) {
+             return client().GetFeature(MakePoint(0, 0));
+           })
+        >> Then([](auto&& feature) {
+             EXPECT_TRUE(feature.has_location())
+                 << "Server returns incomplete feature";
+             auto latitude = feature.location().latitude();
+             auto longitude =
+                 feature.location().longitude();
+
+             if (feature.name().empty()) {
```

Comment:
I don't understand this test. You call `GetFeature()` twice, then it seems that you do an `if` to see which feature you got back? Seems like you should be moving a bunch of these `EXPECT_EQ` up to the first `Then`?